hedera-node/src/test/java/com/hedera/services/txns/crypto/CryptoUpdateTransitionLogicTest.java

```diff
@@ -276,120 +139,120 @@ void rejectsInvalidKey() {
 
 	@Test
 	void rejectsInvalidMemo() {
-		givenTxnCtx(EnumSet.of(AccountCustomizer.Option.MEMO));
+		givenValidTxnCtx();
 		given(validator.memoCheck(MEMO)).willReturn(MEMO_TOO_LONG);
 
 		assertEquals(MEMO_TOO_LONG, subject.semanticCheck().apply(cryptoUpdateTxn));
 	}
 
 	@Test
 	void rejectsInvalidAutoRenewPeriod() {
-		givenTxnCtx();
+		givenValidTxnCtx();
+		givenHappyPathValidator();
 		given(validator.isValidAutoRenewPeriod(any())).willReturn(false);
 
 		assertEquals(AUTORENEW_DURATION_NOT_IN_RANGE, subject.semanticCheck().apply(cryptoUpdateTxn));
 	}
 
-	@Test
-	void acceptsValidTxn() {
-		givenTxnCtx();
-
-		assertEquals(OK, subject.semanticCheck().apply(cryptoUpdateTxn));
-	}
-
 	@Test
 	void rejectsDetachedAccount() {
-		givenTxnCtx();
-		given(ledger.isDetached(TARGET)).willReturn(true);
+		givenValidTxnCtx();
+		givenLoadedAccount();
+		given(target.isDetached()).willReturn(true);
 
-		subject.doStateTransition();
+		assertFailsWith(() -> subject.doStateTransition(), ACCOUNT_EXPIRED_AND_PENDING_REMOVAL);
 
-		verify(txnCtx).setStatus(ACCOUNT_EXPIRED_AND_PENDING_REMOVAL);
+		verifyNeverExecuteBusinessLogic();
```

Comment:
nit: extracting the repeated verification logic is great, I would suggest including the assertFailsWith() in the method, which could look like` thenWillFailWith(SOME_RESPONCE_CODE)`. If the author prefers to not modify the method - its name should probably be renamed "verifyNeverExecute**s**BusinessLogic().